src/renderer/lib/validate.ts

```diff
@@ -11,7 +11,7 @@ export const isValidIpAddress = (ip: string) => {
   return chunks.map((n) => parseInt(n)).every((n) => n >= 0 && n <= 255);
 };
 
-export const isValidIpAndPort = (ip: string) => {
+export const validateIpAndPort = (ip: string): string | boolean => {
```

Comment:
```suggestion
export const validateIpAndPort = (ip: string): string | true => {
```

Reply:
this doesn't work because isValidIpAddress returns a bool.